profiles/fscloud/variables.tf

```diff
@@ -84,18 +80,22 @@ variable "configuration" {
   default = null
 }
 
-variable "key_protect_key_crn" {
+variable "kms_key_crn" {
   type        = string
-  description = "(Optional) The root key CRN of a Key Management Service like Key Protect or Hyper Protect Crypto Service (HPCS) that you want to use for disk encryption. If `null`, database is encrypted by using randomly generated keys. See https://cloud.ibm.com/docs/cloud-databases?topic=cloud-databases-key-protect&interface=ui#key-byok for current list of supported regions for BYOK"
-  default     = null
+  description = "The root key CRN of a Hyper Protect Crypto Service (HPCS) that you want to use for disk encryption. If `null`, database is encrypted by using randomly generated keys. See https://cloud.ibm.com/docs/cloud-databases?topic=cloud-databases-hpcs&interface=ui for more information on integrating HPCS with PostgreSQL database."
```

Comment:
Remove the line `If null, database is encrypted by using randomly generated keys.` - this is not FSCloud compliant - a HyperProtect key is required input here